services/cd-service/pkg/repository/repository.go

```diff
@@ -1517,7 +1519,25 @@ func (s *State) GetEnvironmentApplicationLocks(environment, application string)
 		return result, nil
 	}
 }
-
+func (s *State) GetEnvironmentTeamLocks(environment, team string) (map[string]Lock, error) {
+	base := s.GetTeamLocksDir(environment, team)
+	if entries, err := s.Filesystem.ReadDir(base); err != nil {
+		return nil, err
+	} else {
+		result := make(map[string]Lock, len(entries))
+		for _, e := range entries {
+			if !e.IsDir() {
+				return nil, fmt.Errorf("error getting application locks: found file in the locks directory. run migration script to generate correct metadata")
```

Comment:
```suggestion
				return nil, fmt.Errorf("error getting team locks: found file in the locks directory. run migration script to generate correct metadata")
```
Also, what is the migration script?